Dockerfile

```diff
@@ -5,9 +5,9 @@ RUN mkdir /code && \
     apt-get install -y --no-install-recommends gcc libc6-dev libc-dev libssl-dev && \
     rm -rf /var/lib/apt/lists/*
 WORKDIR /code
-COPY requirements.txt /code/
-COPY requirements-test.txt /code/
+COPY base/requirements.txt /code/
```

Comment:
Sorry for the confusion... When we discussed the redundancy of `requirements/requirements-<type>.txt`, what I meant by moving to a `requirements` directory was to make the `requirements` directory and rename to something like: `requirements.txt` -> `base.txt`, `requirements-dev.txt` -> `dev.txt`, and `requirements-test.txt` -> `test.txt` and group them in `requirements`. I'm not locked into any structure, but `base/` might get confusing.